### PR TITLE
Daily overtime notifier

### PR DESCRIPTION
**Pull request by pewniak747**
This PR removes the daily email notification when employee worked more than 8h a day.

Instead, an email is sent when employee worked more than:
* 8h on monday
* 16h until tuesday (inclusive)
* 24h until wednesday (inclusive)
* 32h until thursday (inclusive)
* 40h until friday (inclusive)

The email includes the amount of accumulated overtime.

### CODE REVIEW

File: spec/daily_notifier_spec.rb [hunk 1]
```diff
@@ -18,91 +18,111 @@
...
 
-      let(:input_time_struct) do
-        [nil, nil, worked_time_in_ms , nil, nil, nil, nil, worked_time_in_ms]
-      end
-
-      context 'user worked more than 8 hours' do
-        let(:worked_time_in_ms) { eight_hours_in_ms + 1 }
+      context 'user worked overtime up until today' do
+        let(:input_time_struct) do
+          [working_day_in_ms, working_day_in_ms, working_day_in_ms+1, nil, nil, nil, nil, 3*working_day_in_ms+1]
```
Comment (marcinkornek) [inline spec/daily_notifier_spec.rb]: Why don't you use spaces between `*`, `+` etc? To be more readable that it's one element in array?

**Comment by marcinkornek** [comment]
looks fine to me 👍  btw. good idea to change this ;)

File: spec/daily_notifier_spec.rb (hunk 1 above)
Comment (pewniak747) [inline spec/daily_notifier_spec.rb]: No, I just missed this in the refactoring pass ;) Thanks, I'll add spaces around operators.

File: Rakefile [hunk 2]
```diff
@@ -32,3 +32,7 @@ task :environment do
     Pony.options[:via_options][:location] = File.expand_path('../tmp/letter_opener', __FILE__)
   end
 end
+
+task console: :environment do
+  binding.pry
```
Comment (angouleme) [inline Rakefile]: 😱

Comment (pewniak747) [inline Rakefile]: this one's intentional, we want to start pry console with this rake task 😄  😱

**Pull request merged**